src/Gamecube_Keyboard.h

```diff
@@ -0,0 +1,97 @@
+#ifndef GCKEYMAP_H_INCLUDED
+#define GCKEYMAP_H_INCLUDED
+
+#define GCKEY_NONE           0x00
```

Comment:
How about this naming schema? `NINTENDO_GC_KEY_NONE`
This would be longer, but more common to our other definitions. At least the GC and KEY should be separated with an underscore.